pkgs/top-level/all-packages.nix

```
@@ -15793,6 +15793,8 @@ with pkgs;
     llvmPackages = llvmPackages_latest;
   };
 
+  refurb = callPackage ../development/tools/refurb { python3 = python310; };
```

It's the default `python3` version already.
```suggestion
  refurb = callPackage ../development/tools/refurb { };
```

---

Thanks, I took all your suggestions in account. For some reason, I couldn't commit suggestions, so I copied the changes manually :(